### DISCUSSION TITLE
Limitations and proxy

### DISCUSSION THREAD

**Discussion by pycharmer1221** [JoMingyu/google-play-scraper#170] [Q&A]
Is there any limitations of number of apis call per day or per month before play store blocks.

Also is there any way i can assign proxy server so that google can not block.

**Comment by Arsh1219** [comment]
Same question. how often we hit the API call before getting blocked?

**Comment by SukebeTako** [comment]
I'm also curious, that there is the limit of reviews or information that can be taken from Google Playstore using this library?
